nadabot/models.py

```diff
@@ -0,0 +1,187 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from accounts.models import Account
+
+
+class ProviderStatus(models.TextChoices):
+    PENDING = "Pending", "Pending"
+    ACTIVE = "Active", "Active"
+    DEACTIVATED = "Deactivated", "Deactivated"
+
+
+class NadabotRegistry(models.Model):
+    id = models.OneToOneField(
+        Account,
+        related_name="registry_id",
+        on_delete=models.CASCADE,
+        primary_key=True,
+        help_text=_("Nadabot registry id."),
+    )
+    owner = models.ForeignKey(
+        Account,
+        on_delete=models.CASCADE,
+        related_name="nadabot",
+        null=False,
+        help_text=_("Nadabot Registry owner."),
+    )
+    default_human_threshold = models.PositiveIntegerField(
+        _("default human threshold"),
+        default=0,
+        help_text=_("default human threshold."),
+    )
+    created_at = models.DateTimeField(
+        _("created at"),
+        null=False,
+        help_text=_("Registry creation date."),
+    )
+    updated_at = models.DateTimeField(
+        _("updated at"),
+        help_text=_("Registry last update date."),
+    )
+    admins = models.ManyToManyField(
+        Account,
+        related_name="admin_registry",
```

Comment:
i would just make this consistent with the `owner` related_name, e.g. `nadabot_admin_registries`